mlflow/store/tracking/file_store.py

```diff
@@ -429,7 +429,19 @@ def delete_experiment(self, experiment_id):
                 databricks_pb2.RESOURCE_DOES_NOT_EXIST,
             )
         experiment = self._get_experiment(experiment_id)
-        experiment._set_last_update_time(get_current_time_millis())
+        experiment._lifecycle_stage = LifecycleStage.DELETED
+        deletion_time = get_current_time_millis()
+        experiment._set_last_update_time(deletion_time)
+        runs = self._list_run_infos(experiment, experiment_id)
+        for run in runs:
+            run_id = run.info.run_id
+            run_info = self._get_run_info(run_id)
+            if run_info is None:
+                raise MlflowException(
+                    "Run '%s' metadata is in invalid state." % run_id, databricks_pb2.INVALID_STATE
+                )
```

Comment:
I think we don't need to raise error here but printing warning instead.

Reply:
> I think we don't need to raise error here but printing warning instead.

Done.